SignalMessaging/categories/UIView+OWS.m

```diff
@@ -487,4 +487,21 @@ - (void)logHierarchyUpwardLaterWithLabel:(NSString *)label
 
 @end
 
+#pragma mark -
+
+@implementation UIScrollView (OWS)
+
+- (void)applyScrollViewInsetsFix
+{
+    // Fix a bug that only affects iOS 11.0.x and 11.1.x.
```

Comment:
Am I right in understanding that we should use this for any scroll view that:

* Could be the bottom-most scroll view of a given view.
* Isn't the root view for its view controller?

In my testing, it looks like we also want to apply to this 11.2.  If we don't, we end up with too much content inset.  Let's discuss?

Reply:
This fix, as I recall, was not to fix the *amount* of content offset, but to fix the weird animation that happens when using the interactive pop gesture in iOS11.0-11.1.

Now that it's centralized, we can at least elaborate the comment in one place, as the current one's pretty opaque.

Reply:
Improved the comment.